core/vm/evm.go

```diff
@@ -536,26 +537,43 @@ func (evm *EVM) TobinTransfer(db StateDB, sender, recipient common.Address, gas
 
 	if amount.Cmp(big.NewInt(0)) != 0 {
 		reserveAddress, err := GetRegisteredAddressWithEvm(params.ReserveRegistryId, evm)
```

Comment:
lacking a bit of context here, but can we have a func in `contract_comm` instead of inlining `getOrComputeTobinTax()` here

Reply:
I don't think so due to circular dependencies (`contract_comm` depends on `core/vm`)